src/Slices/ResourceDiscovery/UI/DiscoveredResourcesTable.tsx

```diff
@@ -62,7 +62,7 @@ export const DiscoveredResourcesTable: React.FC<Props> = ({
     <Table {...props} variant={TableVariant.compact}>
       <Thead>
         <Tr>
-          <Th></Th>
+          <Th aria-hidden></Th>
```

Comment:
same here, I know that it wasn't changed by you, but I just noticed that so wanted to point it out :)